### PR TITLE
NIAD-703 Create methods for sandbox test environment

### PR DESCRIPTION
**Pull request by graham-andrew**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: docker/service/src/main/java/uk/nhs/adaptors/scr/clients/SandboxSpineClient.java [hunk 1]
```diff
@@ -2,16 +2,22 @@
...
 import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.springframework.http.HttpStatus.ACCEPTED;
 import static org.springframework.http.HttpStatus.OK;
+import static uk.nhs.adaptors.scr.clients.SpineHttpClient.CONTENT_LOCATION_HEADER;
+import static uk.nhs.adaptors.scr.clients.SpineHttpClient.RETRY_AFTER_HEADER;
 
 public class SandboxSpineClient implements SpineClientContract {
 
     private static final String EVENT_LIST_QUERY_RESPONSE = "mock-spine/QUPC_IN200000SM04/success.xml";
+    private static final String POST_SCR_PROCESSED_RESPONSE = "mock-spine/MCCI_IN010000UK13/success.xml";
```
Comment (dawidskainos) [inline docker/service/src/main/java/uk/nhs/adaptors/scr/clients/SandboxSpineClient.java:20]: Can we rename it? Maybe UPLOAD_SCR_POLLING_RESPONSE?

File: docker/service/src/main/java/uk/nhs/adaptors/scr/clients/SandboxSpineClient.java [hunk 2]
```diff
@@ -20,13 +26,18 @@ public Response sendAcsData(String requestBody) {
 
     @Override
     public Response sendScrData(String requestBody, String nhsdAsid, String nhsdIdentity, String nhsdSessionUrid) {
-        return null;
+        Header[] headers = {
+            new BasicHeader(CONTENT_LOCATION_HEADER, ""),
+            new BasicHeader(RETRY_AFTER_HEADER, "1000")
```
Comment (dawidskainos) [inline docker/service/src/main/java/uk/nhs/adaptors/scr/clients/SandboxSpineClient.java:31]: 1000 means that app will start polling after 1000ms. Maybe we could use a smaller value to improve sandbox users experience? Maybe 300?

**Review by dawidskainos — CHANGES_REQUESTED** [review]

File: docker/service/src/main/java/uk/nhs/adaptors/scr/clients/SandboxSpineClient.java (hunk 1 above)
Comment (graham-andrew) [reply docker/service/src/main/java/uk/nhs/adaptors/scr/clients/SandboxSpineClient.java]: Renamed.

File: docker/service/src/main/java/uk/nhs/adaptors/scr/clients/SandboxSpineClient.java (hunk 2 above)
Comment (graham-andrew) [reply docker/service/src/main/java/uk/nhs/adaptors/scr/clients/SandboxSpineClient.java]: Time reduced to 300ms.

**Review by dawidskainos — APPROVED** [review]

**Pull request merged**